src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -201,8 +211,10 @@ const UsersFragment = ({
         members.push(profile?.myProfile?.user_id, item?.user_id || item?.userId);
         const processGetBlock = await checkUserBlock(sendData);
         if (!processGetBlock.data.data.blocked && !processGetBlock.data.data.blocker) {
+          setLoadingDM(false);
           return createSignChat(members, item);
         }
+        setLoadingDM(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `setLoadingDM(false)` before returning from the function in certain conditions is a good practice. It ensures that the loading state is reset even if the function exits early.
```diff
+           setLoadingDM(false);
            return createSignChat(members, item);
+          }
+         setLoadingDM(false);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->